### PR TITLE
Week 1 folder & regex research added

### PR DESCRIPTION
**Pull request by vlbee**

### CODE REVIEW

File: week-1/RegexResearch.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+https://www.regexbuddy.com/   - good explanations
+https://regexr.com/ 
+https://www.regexpal.com/
+https://regex101.com/
+http://eloquentjavascript.net/09_regexp.html # regex chapter of eloquent javascript
+https://www.w3schools.com/jsref/jsref_obj_regexp.asp
+
+
+
+## Presentation Link with Repl examples
```
Comment (lucymk) [inline week-1/RegexResearch.md]: I wonder if it might be better to put the presentation link at the top, to allow for easier access? 😃

**Review by lucymk — CHANGES_REQUESTED** [review]
I wonder if it might be better to put the presentation link at the top, to allow for easier access? 😃

**Comment by vlbee** [comment]
@lucymk  I moved the presentation link up to the top :)

**Comment by vlbee** [comment]
@lucymk  Also now added link to main Readme.

**Review by lucymk — APPROVED** [review]

**Pull request merged**